io.github.efogdev.mpris-timer.yml

```diff
@@ -0,0 +1,38 @@
+id: io.github.efogdev.mpris-timer
+runtime: org.gnome.Platform
+runtime-version: "47"
+sdk: org.gnome.Sdk
+command: mpris-timer
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=~/.var/app:create
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
do you need this? If it is gtk4, it should go via the portal.

Reply:
The app might be started without GUI, I don't create GTK application in that case as a performance optimization

Reply:
also fixed